packages/grafana-data/src/text/sanitize.ts

```diff
@@ -56,13 +56,29 @@ const sanitizeTextPanelWhitelist = new xss.FilterXSS({
  */
 export function sanitize(unsanitizedString: string): string {
   try {
+    // hook to handle anchor elements with target="_blank"
+    DOMPurify.addHook('afterSanitizeAttributes', function (node) {
+      if (node.tagName === 'A' && node.getAttribute('target') === '_blank') {
+        const currentRel = node.getAttribute('rel') || '';
+        const relValues = new Set(currentRel.split(/\s+/).filter(Boolean));
+
+        relValues.add('noopener');
```

Comment:
*comment(non-blocking):* :speech_balloon: My understanding is that `_blank` should provide behaviour equivalent to `noopener` implicitly / by default.

- SEE: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/rel/noopener

However, I think the "belt and suspenders" approach of explicitly setting that attribute in this case is very clear _(no HTML trivia knowledge needed)_ and does no harm.